src/components/ArrowToggle/README.md

```diff
@@ -115,8 +115,8 @@ return (
 
 ## Properties
 
-| Name      | Description                 |   Type   |  Default   |
-| :-------- | :-------------------------- | :------: | :--------: |
-| className | HTML `class` attribute      | `string` |            |
-| direction | Sets arrowToggle directions | `string` | `"bottom"` |
-| size      | Size arrowToggle in px      | `number` |    `16`    |
+| Name      | Description                          |   Type   |  Default   |
+| :-------- | :----------------------------------- | :------: | :--------: |
+| className | HTML `class` attribute               | `string` |            |
+| direction | Used to set direction of arrowToggle | `string` | `"bottom"` |
```

Comment:
Looks like it needs an article before `direction`

Reply:
Added it!